packages/babel-traverse/src/scope/index.ts

```diff
@@ -1119,7 +1119,10 @@ export default class Scope {
 
     const declarator = variableDeclarator(id, init);
     const len = declarPath.node.declarations.push(declarator);
-    path.scope.registerBinding(kind, declarPath.get("declarations")[len - 1]);
+    path.scope.registerBinding(
+      ["using", "await using"].includes(kind) ? "const" : (kind as BindingKind),
+      declarPath.get("declarations")[len - 1],
```

Comment:
Also, could you add a test for this change? (and maybe split it to another PR if it fixes a bug :P)

Reply:
Ah! I should fix it another way.
[`44627e5` (#15822)](https://github.com/babel/babel/pull/15822/commits/44627e54fdb862a31148cd78c6fa62d89d8312d0)